uiowa/uids
# Add a build process for generating CSS files and instructions for how to use them

Relates to https://github.com/uiowa/uids/issues/908. 

# How to test

- `yarn production`
- `npx http-server ./www`
- Go to the Introduction page and confirm that CSS files are listed using the version number from `package.json`. 
- Go to the Brand Bar docs page and confirm that the CSS  files are listed using the version number from `package.json`.
